src/components/shared/cellRenderers/glimmerRenderer/glimmerRenderer.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from "react";
+
+import { svgs } from 'utilities';
+import { Svg } from 'components/shared/svg/svg';
+
+import '../cellRenderer.css';
+
+export const GlimmerRenderer = (props) => (
+  props.value
+  ? <Svg path={svgs.glimmer} className="glimmer-icon" />
```

Comment:
I think this ternary statement can be combined in single line.

Reply:
If I recall, it went over 90 characters.... and I didn't want you to ask me to split it into different lines. :)